apps/alphabet_mobile/App.tsx

```diff
@@ -0,0 +1,114 @@
+/**
```

Comment:
Remove boilerplate.

Reply:
Just add a TODO for now. This PR is really just about the scaffolding anyway, not the actual app.